i18n/harvest.i18n.dart

```diff
@@ -4,11 +4,11 @@ extension Localization on String {
 
 static var _t = Translations.byLocale("en_us") +
     {
-      "es_es": {
-        "Transaction hash: %s": "Hash de transacción",
-        "Success!": "Éxito!",
+      "pt_br": {
+        "Transaction hash: %s": "Hash de transação",
+        "Success!": "Sucesso!",
         "Plant Seeds": "Plantar Seeds",
-        "Plant amount": "Monto a plantar",
+        "Plant amount": "Quantiade a plantar",
```

Comment:
Here should be "Quantidade a plantar" little typo, miss the letter 'd'. @juliolrmonteiro

Reply:
Fixed on latest commit, thanks!